### PR TITLE
AQCU-1729: move gsplot install to repgen container

### PR DESCRIPTION
**Pull request by mhines-usgs**
must be merged along with https://github.com/USGS-CIDA/docker-aqcu-rserve/pull/26

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+  wget -O /tmp/install/installPackages.R https://raw.githubusercontent.com/USGS-R/repgen/${REPGEN_VERSION}/inst/extdata/installPackages.R && \
+  wget -O /tmp/install/gsplot_description_dir/DESCRIPTION https://raw.githubusercontent.com/USGS-R/gsplot/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}/DESCRIPTION
+
+# Install GSplot
+RUN mkdir ${RSERVE_HOME}/R_libs && \
+  mkdir ${RSERVE_HOME}/work && \
+  cd /tmp/install/gsplot_description_dir && \
+  Rscript /tmp/install/installPackages.R && \
+  Rscript -e "library(devtools);install_url('https://github.com/USGS-R/gsplot/archive/v${GSPLOT_VERSION:-$GSPLOT_VERSION_DEFAULT}.zip', dependencies = F)" && \
+  rm -rf /tmp/install
```
Comment (mhines-usgs) [inline Dockerfile]: guessing i should move this to the bottom then i could get rid of the extra wget for the installPackages.R file

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,24 @@
-FROM cidasdpdasartip.cr.usgs.gov:8447/aqcu/rserve:latest
+#FROM cidasdpdasartip.cr.usgs.gov:8447/aqcu/rserve:latest
+FROM aqcu-rserve:0.0.1
 
 ENV REPGEN_VERSION=master
+ENV GSPLOT_VERSION_DEFAULT 0.8.1
```
Comment (zmoore-usgs) [inline Dockerfile]:
```suggestion
ENV GSPLOT_VERSION_DEFAULT=0.8.1
```

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,24 @@
-FROM cidasdpdasartip.cr.usgs.gov:8447/aqcu/rserve:latest
+#FROM cidasdpdasartip.cr.usgs.gov:8447/aqcu/rserve:latest
+FROM aqcu-rserve:0.0.1
```
Comment (zmoore-usgs) [inline Dockerfile]: I think this will need to change back to cidasdpdasartip before merging

Comment (mhines-usgs) [reply Dockerfile]: woops, yes, i meant to remove that!

**Pull request merged**